### PR TITLE
Michael dratch/712 error message if map fails to load

### PR DESCRIPTION
**Pull request by Michael-Dratch**
This PR is for this task: Show error message if map fails to load #712

https://github.com/CodeForPhilly/clean-and-green-philly/issues/712

To view the error state go to the find properties page, open dev tools, and block the URL from google storage to simulate a failed API call.

image

On page reload, this error message will be displayed on the property details panel.

image

I think the automatic prettier formatting makes it seem like a lot of lines of code have been changed. But most of these lines are minor changes in formatting. If it would be better for the PR I can redo do these changes with out any of the formatting changes and resubmit.

### CODE REVIEW

File: src/app/find-properties/[[...opa_id]]/page.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ const MapPage = ({ params }: MapPageProps) => {
   const [featureCount, setFeatureCount] = useState<number>(0);
   const [currentView, setCurrentView] = useState<BarClickOptions>("detail");
   const [loading, setLoading] = useState(true);
+  const [loadingError, setLoadingError] = useState(false);
```
Comment (CodeWritingCow) [inline src/app/find-properties/[[...opa_id]]/page.tsx:36]: Can we rename `loadingError` and its setter to something like `hasLoadingError` and `setHasLoadingError`?

This variable has a boolean value, but its current name can give the wrong impression. `loadingError` sounds like a name for something else, such as an error response object.

Comment (CodeWritingCow) [reply src/app/find-properties/[[...opa_id]]/page.tsx:36]: This variable has a boolean value, but its current name can give the wrong impression. `loadingError` sounds like a name for something else, such as an error response object. (I realize the line above has a boolean variable named `const loading`, but let's not follow that example.)

File: src/components/PropertyDetailSection.tsx [hunk 2]
```diff
@@ -50,6 +50,7 @@ interface PropertyDetailSectionProps {
   featuresInView: MapGeoJSONFeature[];
   display: "detail" | "list";
   loading: boolean;
+  loadingError: boolean;
```
Comment (CodeWritingCow) [inline src/components/PropertyDetailSection.tsx:53]: Please see previous comment about renaming the variable.

**Review by CodeWritingCow — CHANGES_REQUESTED** [review]
Hi @Michael-Dratch I tested your PR. The error message appears correctly if the map API call fails. Once you make the changes mentioned in my comments, I'll approve and merge your PR. Thanks!

**Comment by Michael-Dratch** [comment]
Hi @CodeWritingCow, thanks for the feedback. I just made those requested updates and pushed the changes.

**Review by CodeWritingCow — APPROVED** [review]
@Michael-Dratch updated PR looks good. Thanks for your contribution!

**Pull request merged**